packages/boutique/frontend/src/components/navigation.tsx

```diff
@@ -11,12 +11,12 @@ const NAV_LINKS = [
     to: '/products',
     name: 'Products',
     icon: <Products />
-  },
-  {
-    to: '/orders',
-    name: 'Orders',
-    icon: <Orders />
   }
+  // {
+  //   to: '/orders',
```

Comment:
is this needed?

Reply:
It will probably be needed. Just commenting it out for now.